.github/workflows/change-plugin-scope.yml

```
@@ -0,0 +1,113 @@
+# Description:
+# Change plugin scope.
+# It uses action grafana/plugin-ci-workflows/actions/plugins/publish/promote-plugin
+# to change plugin scope to universal.
+# The following inputs should be provided:
+# - environment: Target environment (dev, ops, prod)
+# - scopes: Scopes to change to. Can be 'universal', 'grafana_cloud', 'grafana_cloud_org' or 'grafana_cloud_instance'.
+#           Multiple values can be provided separated by commas.
+# - plugin_id: Plugin ID to change scope
+# - plugin_version: Plugin version to change scope
+#
+
+name: Plugins - Change Plugin Scope
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Target environment'
+        required: true
+        default: 'dev'
+        type: choice
+        options:
+          - dev
+          - ops
+          - prod
+      plugin_id:
+        description: 'Plugin ID to change scope'
+        required: true
```

This is not required anymore since the workflow has logic to handle a default value when the input is empty

```suggestion
        required: false
```